sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -147,12 +158,12 @@ case class Invoke(
     }
```

Comment:
Here `${ev.value}` will be `null` only when `ctx.defaultValue(javaType)` is `null`. There is a `postNullCheck` to do the extra check for this case.